### ISSUE TITLE
Linear Transducer

### ISSUE THREAD

**Issue by ghost** [USgHIFU/USgHIFU-qt#16]
MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz)  is good for further contribution and development? Or what can you recommend with this program?

**Comment by audrew** [comment]
which Piezoelectric Transducers works with it?

**Comment by USgHIFU** [comment]
> MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz) is good for further contribution and development?
Yes, almost any US imaging probe can be integrated into this system.

**Comment by USgHIFU** [comment]
> which Piezoelectric Transducers works with it?

You mean high-intensity focused ultrasound phased array?

**Comment by audrew** [comment]
> 

Yes it can be phased array, its just that it operates on higher frequencies from 3 to 10 MHz, I also would be glad to have instructions of how to use software or guide to how to find such instructions if you will. I am happy to collaborate and develop this technology further

**Comment by audrew** [comment]
> MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz) is good for further contribution and development? Or what can you recommend with this program?

as my knowledge today MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz) works good in diagnostics only

**Comment by audrew** [comment]
> > which Piezoelectric Transducers works with it?
> 
> You mean high-intensity focused ultrasound phased array?

Yes it can be phased array, its just that it operates on higher frequencies from 3 to 10 MHz and it's also used for treatment and not only diagnostics. I also would be glad to have instructions of how to use software or guide to how to find such instructions if you will. I am happy to collaborate and develop this technology further

**Comment by ghost** [comment]
> > MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz) is good for further contribution and development?
> > Yes, almost any US imaging probe can be integrated into this system.

Does vivid 5 Linear Transducer  4-10 MHz probe has TEE connector adapter in order to work with any of computer ports such as hdmi, usb or vda? I can only find vivid 5 computer machines available with vivid 5 software installed.. Otherwise I did find wifi transducer which works with mobile app. If one of these choices goes along with this software?

**Comment by USgHIFU** [comment]
> > > MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz) is good for further contribution and development?
> > > Yes, almost any US imaging probe can be integrated into this system.
> 
> Does vivid 5 Linear Transducer 4-10 MHz probe has TEE connector adapter in order to work with any of computer ports such as hdmi, usb or vda? I can only find vivid 5 computer machines available with vivid 5 software installed.. Otherwise I did find wifi transducer which works with mobile app. If one of these choices goes along with this software?

Sorry that I don't understand what the big picture of your project is. Could you please tell more if possible?

**Comment by ghost** [comment]
> > > > MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz) is good for further contribution and development?
> > > > Yes, almost any US imaging probe can be integrated into this system.
> > 
> > 
> > Does vivid 5 Linear Transducer 4-10 MHz probe has TEE connector adapter in order to work with any of computer ports such as hdmi, usb or vda? I can only find vivid 5 computer machines available with vivid 5 software installed.. Otherwise I did find wifi transducer which works with mobile app. If one of these choices goes along with this software?
> 
> Sorry that I don't understand what the big picture of your project is. Could you please tell more if possible?

High-intensity focused ultrasound is a non-invasive therapeutic technique that uses non-ionizing ultrasonic waves to heat or ablate tissue. For reasons such as skin tightening, increase the flow of blood or lymph, prostate cancers and such.

**Comment by USgHIFU** [comment]
> > > > > MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz) is good for further contribution and development?
> > > > > Yes, almost any US imaging probe can be integrated into this system.
> > > 
> > > 
> > > Does vivid 5 Linear Transducer 4-10 MHz probe has TEE connector adapter in order to work with any of computer ports such as hdmi, usb or vda? I can only find vivid 5 computer machines available with vivid 5 software installed.. Otherwise I did find wifi transducer which works with mobile app. If one of these choices goes along with this software?
> > 
> > 
> > Sorry that I don't understand what the big picture of your project is. Could you please tell more if possible?
> 
> High-intensity focused ultrasound is a non-invasive therapeutic technique that uses non-ionizing ultrasonic waves to heat or ablate tissue. For reasons such as skin tightening, increase the flow of blood or lymph, prostate cancers and such.

Of course HIFU, what I mean is the indication, and the system setup.

**Comment by ghost** [comment]
> > > > > > MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz) is good for further contribution and development?
> > > > > > Yes, almost any US imaging probe can be integrated into this system.
> > > > 
> > > > 
> > > > Does vivid 5 Linear Transducer 4-10 MHz probe has TEE connector adapter in order to work with any of computer ports such as hdmi, usb or vda? I can only find vivid 5 computer machines available with vivid 5 software installed.. Otherwise I did find wifi transducer which works with mobile app. If one of these choices goes along with this software?
> > > 
> > > 
> > > Sorry that I don't understand what the big picture of your project is. Could you please tell more if possible?
> > 
> > 
> > High-intensity focused ultrasound is a non-invasive therapeutic technique that uses non-ionizing ultrasonic waves to heat or ablate tissue. For reasons such as skin tightening, increase the flow of blood or lymph, prostate cancers and such.
> 
> Of course HIFU, what I mean is the indication, and the system setup.

Could you provide more information which machine or transducer probe was used when developing this repo?

**Comment by USgHIFU** [comment]
> 

MyLab 90

**Comment by USgHIFU** [comment]
> > > > > > > MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz) is good for further contribution and development?
> > > > > > > Yes, almost any US imaging probe can be integrated into this system.
> > > > > 
> > > > > 
> > > > > Does vivid 5 Linear Transducer 4-10 MHz probe has TEE connector adapter in order to work with any of computer ports such as hdmi, usb or vda? I can only find vivid 5 computer machines available with vivid 5 software installed.. Otherwise I did find wifi transducer which works with mobile app. If one of these choices goes along with this software?
> > > > 
> > > > 
> > > > Sorry that I don't understand what the big picture of your project is. Could you please tell more if possible?
> > > 
> > > 
> > > High-intensity focused ultrasound is a non-invasive therapeutic technique that uses non-ionizing ultrasonic waves to heat or ablate tissue. For reasons such as skin tightening, increase the flow of blood or lymph, prostate cancers and such.
> > 
> > 
> > Of course HIFU, what I mean is the indication, and the system setup.
> 
> Could you provide more information which machine or transducer probe was used when developing this repo?

MyLab 90

**Comment by ghost** [comment]
> 

Do you use whole machine with it or just linear transducer probe with adapters?

**Comment by ghost** [comment]
> > > > > > > > MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz) is good for further contribution and development?
> > > > > > > > Yes, almost any US imaging probe can be integrated into this system.
> > > > > > 
> > > > > > 
> > > > > > Does vivid 5 Linear Transducer 4-10 MHz probe has TEE connector adapter in order to work with any of computer ports such as hdmi, usb or vda? I can only find vivid 5 computer machines available with vivid 5 software installed.. Otherwise I did find wifi transducer which works with mobile app. If one of these choices goes along with this software?
> > > > > 
> > > > > 
> > > > > Sorry that I don't understand what the big picture of your project is. Could you please tell more if possible?
> > > > 
> > > > 
> > > > High-intensity focused ultrasound is a non-invasive therapeutic technique that uses non-ionizing ultrasonic waves to heat or ablate tissue. For reasons such as skin tightening, increase the flow of blood or lymph, prostate cancers and such.
> > > 
> > > 
> > > Of course HIFU, what I mean is the indication, and the system setup.
> > 
> > 
> > Could you provide more information which machine or transducer probe was used when developing this repo?
> 
> MyLab 90

Do you use MyLab 90 machine or laptop with linear probe and adapter from TEE to laptop?

**Comment by ghost** [comment]
> > > > > > > > MINDRAY 7L4s Linear Transducer (FBW: 4-10 MHz) is good for further contribution and development?
> > > > > > > > Yes, almost any US imaging probe can be integrated into this system.
> > > > > > 
> > > > > > 
> > > > > > Does vivid 5 Linear Transducer 4-10 MHz probe has TEE connector adapter in order to work with any of computer ports such as hdmi, usb or vda? I can only find vivid 5 computer machines available with vivid 5 software installed.. Otherwise I did find wifi transducer which works with mobile app. If one of these choices goes along with this software?
> > > > > 
> > > > > 
> > > > > Sorry that I don't understand what the big picture of your project is. Could you please tell more if possible?
> > > > 
> > > > 
> > > > High-intensity focused ultrasound is a non-invasive therapeutic technique that uses non-ionizing ultrasonic waves to heat or ablate tissue. For reasons such as skin tightening, increase the flow of blood or lymph, prostate cancers and such.
> > > 
> > > 
> > > Of course HIFU, what I mean is the indication, and the system setup.
> > 
> > 
> > Could you provide more information which machine or transducer probe was used when developing this repo?
> 
> MyLab 90

Could you please provide better explanation? Do you use MyLab 90 machine or probe with adapter? I am getting wifi 10 mhz linear probe, do you think that will work if I will integrate wifi module into this software?
